pyproject.toml

```diff
@@ -1,3 +1,86 @@
+[tool.poetry]
+name = "fedora_messaging"
+version = "3.4.1"
+description = "A set of tools for using Fedora's messaging infrastructure"
+authors = [
+    "Fedora Infrastructure <admin@fedoraproject.org>"
+]
+license = "GPLv2+"
+readme = "README.rst"
+repository = "https://github.com/fedora-infra/fedora-messaging"
+maintainers = ["Fedora Infrastructure Team"]
+classifiers = [
+        "Development Status :: 5 - Production/Stable",
+        "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+    ]
+keywords = ["fedora"]
+packages = [
+    { include = "fedora_messaging" },
+]
+include = [
+    { path = "docs", format = "sdist" },
+		{ path = "CODEWNERS", format = "sdist"},
+		{ path = "configs", format = "sdist"},
+		{ path = "*.service", format = "sdist"},
+]
+
+[tool.poetry.dependencies]
+python = ">=3.9"
```

Comment:
We used to be compatible with python 3.6. I think we still want to keep that as it's the default on EL8.

Reply:
Hopefully fixed although cannot wait for dropping python 3.6 :)
Fixed for main dependencies, still working on dev-dependencies and docs-dependencies.